docs/tutorials/using-pixy-camera.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Using the CMUcam5 Pixy for LEGO Mindstorms
+group: hardware-extras
+author: "@KWSmit"
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+The CMUcam5 Pixy camera is available in a version designed for
+LEGO Mindstorms. In this tutorial we explain how to use it with ev3dev
+and python.
+
+General information about Pixy for LEGO Mindstorms can be found on
+Charmed Lab's [wiki-page](http://cmucam.org/projects/cmucam5/wiki/LEGO_Wiki).
+
+## What you need
+
+* LEGO Mindstorms EV3.
+* CMUcam5 Pixy for LEGO Mindstorms, connected to the EV3 with the cable
+delivered with your camera. If you have a regular version of CMUcam5 Pixy
+you will need a special Pixy adapter for EV3.
+
+**Note:** the Pixy camera comes with it's own tool: PixyMon. This tool helps
```

Comment:
Today I incidentally found out that you don't need to unplug the camera from your PC, you only need to be sure the PixyMon tool is not running! I will change the tutorial for this.